notebooks/benchmark-format-template.ipynb

```diff
@@ -163,7 +165,9 @@
    },
    "outputs": [],
    "source": [
-    "final_array"
+    "%%time\n",
+    "assert final_array.shape == (298271231,)\n",
+    "final_array.mean()"
```

Comment:
@abarciauskas-bgse, were you thinking of concatenating the `h_ph` (photon height) data variable across all 5 ATL03 files into a single 1D array just like xarray? Or some other shape?

Reply:
Ah ok, I see that your new commit at c9ac2100e38253334dba2f0e1b7d5a359a6be8d0, let me reconcile those together later (after lunch).

Reply:
Thanks @weiji14 I was just about to comment about that!

Reply:
Ok, have synced up with the stuff from commit e9aedc4905a85d2349a3c272cf4f6b8399089877, going to squash merge this into main now!